kyuubi-server/pom.xml

```diff
@@ -63,7 +63,33 @@
         <dependency>
             <groupId>org.apache.hadoop</groupId>
             <artifactId>hadoop-client-runtime</artifactId>
-            <scope>runtime</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>org.apache.hive</groupId>
+            <artifactId>hive-metastore</artifactId>
```

Comment:
cc @pan3793 add some new hive dependency since we need connect to hive metastore

Reply:
The change LGTM, only 3 hive jars are added. But one question, should we support multi hive versions?

Reply:
Sounds reasonable. Let me check whether the current code can work with different versions of Hive metastore.

If it works, maybe we can implement an isolated HiveMetaStoreClient later when we need to use other HiveMetaStoreClient methods. 
If not, I will add a new sub-task and complete it.  And we can turn off HiveDelegationToken renewal before this sub-task completes.

Reply:
Verified that `HiveMetaStoreClient#getDelegationToken(java.lang.String, java.lang.String)` can work with Hive version 1.2.2, 2.3.9, 3.1.2.

Reply:
Thanks for works on it.